tests/integration/expected_test_results/search_v1/search-no-filter.json

```diff
@@ -1832,23 +1832,14 @@
       {
```

Comment:
We have those differences because of the bug: we were setting "ingredients_text" with English ingredients while the main language of the product (lang field) was Spanish, so we analyzed (wrongly) ingredients, and set the ingredients analysis fields.

With this correction, we instead set ingredients_text_en, (and not ingredients_text), and we don't analyze ingredients because we don't have them in the main language of the product.

@raphael0202 's PR https://github.com/openfoodfacts/openfoodfacts-server/pull/8855 will then change those tests again: we will detect we have ingredients in English, and analyze them.